### PR TITLE
Restore Windows support

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: libnodegl/log.c [hunk 1]
```diff
@@ -58,6 +58,7 @@ static void default_callback(void *arg, int level, const char *filename, int ln,
     printf("%s[%s] %s:%d %s: %s%s\n", color_start,
            log_strs[level], filename, ln, fn, logline,
            color_end);
+    fflush(stdout);
```
Comment (mbouron-gpsw) [inline libnodegl/log.c:61]: Won't this degrade performance when log level is set to debug on others systems ? Wouldn't scoping it to Windows only be better ?

**Comment by mbouron-gpsw** [comment]
Other than my remark regarding the use of fflush(), LGTM.

File: libnodegl/log.c (hunk 1 above)
Comment (cboesch-gpsw) [reply libnodegl/log.c:61]: Well, the logging itself is in a pipe, other system are already flushing per line. Don't we want to force that specific behavior anyway? I mean, do we really want log to allow buffering many/infinite lines before showing them to the user?

Comment (mbouron-gpsw) [reply libnodegl/log.c:61]: Ok if it does not change anything without a pipe.

**Review by mbouron-gpsw — APPROVED** [review]

**Pull request merged**